materials/section-4/README.md

```diff
@@ -0,0 +1,87 @@
+# Database (Extra Credit)
+
+In this extra credit section we will add a database and integrate it with our app. Storing state in a atom is perfectly fine if you don't want to have persistent data, otherwise you probably need a datastore or database. We will be using _postgres_ as our database. We won't be talking about specific databases in this section, only how you integrate with them (in this case postgres).
+
+### Prerequisite
+
+For this section, you will need docker installed on your machine. Follow [this](https://runnable.com/docker/getting-started/) guide to get started with docker. We won't be discussing docker in any great detail in this section, as this section only explains how to integrate with a database.
+
+### Setup
+
+Once you have docker installed, create a `Dockerfile` inside the `section-2` folder. The `Dockerfile` should look like this:
+
+```
+FROM postgres:10
+ENV POSTGRES_DB clojure_workshop_db
+ADD create_table.sql /docker-entrypoint-initdb.d/
+```
+
+On the last line we copy a sql file (create_table.sql) into our container. This file will create our database table. The file should look like this:
+
+```sql
+CREATE TABLE todo(
+  id serial PRIMARY KEY NOT NULL,
+  name varchar(50) NOT NULL,
+  done BOOL DEFAULT FALSE
```

Comment:
For consistency, might it be an idea to either lowercase this type, or alternatively uppercase the two rows above? 

```suggestion
  done BOOL DEFAULT FALSE
```